owncloudApp/src/main/res/layout/video_preview.xml

```diff
@@ -18,16 +18,12 @@
     android:id="@+id/root"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:keepScreenOn="true"
-    android:background="@color/black">
+    android:keepScreenOn="true">
 
-    <com.google.android.exoplayer2.ui.PlayerView
+    <androidx.media3.ui.PlayerView
         android:id="@+id/video_player"
+        app:show_buffering="always"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        app:controller_layout_id="@layout/custom_playback_control" />
```

Comment:
Just curiosity, with Media3 we don't need a controller layout? Does it provide one by default? 💡

Reply:
Yes, It has the default layout that you see because as you can see the view element is itself `PlayerView`.

The `controller_layout` that was being used earlier was a workaroud , that layout was also the default layout that used to come with player view but it was copied from the source code of Exoplayer and used with added Full Screen In/Out buttons. 

`controller_layout_id` attribute is only used when we want to use custom player view otherwise the default one comes with basic features already.